packages/@coorpacademy-components/src/template/external-course/index.js

```diff
@@ -76,13 +89,19 @@ class ExternalCourse extends React.Component {
           </div>
           <Button
             type="button"
-            disabled={complete.disabled}
-            onClick={this.handleOnClick(complete)}
+            disabled={loading ? true : complete.disabled}
+            onClick={loading ? identity : this.handleOnClick(complete)}
```

Comment:
If it's disabled when loading, no need for the noob function, not?

Reply:
one can never be too careful